pkg/core/namespace.go

```diff
@@ -143,6 +167,33 @@ func (c *client) NamespaceCleanup(options NamespaceCleanupOptions) error {
 	return nil
 }
 
+func (c *client) NamespaceAddCredentials(options NamespaceAddCredentialsOptions) error {
+	namespace := options.NamespaceName
+	secretName := options.SecretName
+	exists, err := c.checkSecretExists(namespace, secretName)
+	if err != nil {
+		return err
+	}
+	if exists {
+		return fmt.Errorf("secret %q already exists", secretName)
```

Comment:
rather than error out, update the secret

Reply:
Do we want this for this command only or for `namespace init` as well? This piece of code is currently shared by both.

Reply:
The newest commit does not change `namespace init` behavior but allows secret updates in `credentials set`.